insolar/mainnet
# OPS-354 run e2e tests on pull requests

**- What I did**
Added GHA workflow which executes e2e [go-autotests](https://github.com/insolar/go-autotests) on current git commit.

**- How I did it**

**- How to verify it**
Run e2e workflow (see [checks](https://github.com/insolar/mainnet/pull/82/checks))
**- Description for the changelog**
